### ISSUE TITLE
USD 24.08 + OCIO 2.3, HdxColorCorrectionTask error in usdview with storm on linux

### ISSUE THREAD

**Issue by pberto** [PixarAnimationStudios/USD#3274]
### Steps to Reproduce
1. Make a build of USD 24.08 build with OCIO 2.3 on Linux
2. OCIO variable is set, usdview is launched. Use soem color profiles like the default SPI ones.
3. when switching the various color profiles in usdview UI, we see totally borked colors in the storm viewport, along with these errors on the cli:

```
ERROR: Usdview encountered an error while rendering.
        Error in 'pxr_v2408__pxrReserved__::HdxColorCorrectionTask::_CreateOpenColorIOShaderCode' at line 562 in file /pxr/imaging/hdx/colorCorrectionTask.cpp : 'Failed verification: ' !texInfo.texName.empty() && !texInfo.samplerName.empty() ''
```

Any pointers?

**Comment by jesschimein** [comment]
Filed as internal issue #USD-10086

**Comment by pberto** [comment]
Would be great to have some feedback about it as it’s something we would like to use.

**Comment by jesschimein** [comment]
Hi!  Unfortunately, it looks like it may be a while before we’ll be able to work on this issue.

**Troubleshooting**
 Here are a few (free!) resources to help you troubleshoot on your own or with the help of other awesome OpenUSD users. If you end up finding a solution, we’d greatly appreciate it if you could comment on this issue with the resolution and close it out.

- Check if this issue has already been fixed by trying to reproduce it on the [dev branch](https://github.com/PixarAnimationStudios/OpenUSD/tree/dev) (see [release cadence](https://openusd.org/release/release_schedule.html)) 
    - Please close the issue with a comment like “Closed” if it’s no longer an issue! 
- Consult the [OpenUSD tutorials](https://openusd.org/release/tut_usd_tutorials.html), [FAQ]s (https://openusd.org/release/tut_usd_tutorials.html), and [API documentation](https://openusd.org/release/apiDocs.html). 
- Check the [ASWF USD Working Group Wiki](https://wiki.aswf.io/display/WGUSD). 
- Consult the [AOUSD Forum](https://forum.aousd.org/) – a searchable community forum; free to join and post. 
    - If you’re having trouble building OpenUSD, don’t hesitate to ask for help in the [Build and Distribution community](https://forum.aousd.org/c/community/build-and-distribution/9) 
- Search the [usd-interest forum](https://groups.google.com/g/usd-interest) – a legacy forum closed to new posts but still a valuable searchable archive. 
- Ask in a [ASWF USD Working Groups meeting](https://www.aswf.io/get-involved/) and/or in the ASWF Slack (#wg-usd).

**Help Us Prioritize**
If you don’t find a solution, there are still a few things you can do to help us get to this more quickly: 
1. Converge multiple related discussions into a single issue so we can see hotspots of need. 
    1. [Search open issues/PRs](https://github.com/PixarAnimationStudios/OpenUSD/issues?q=is%3Aopen+) in this repo.
    2. A good rule-of-thumb is to keep open the issue that captures the most information/discussion/state, and close out others as duplicates.
2. Provide clear reproduction steps and system, package, and build information, if you haven’t already. Comparison images and test assets are also welcome, if applicable. 
3. Take a stab at contributing a PR if you have an idea of how to solve this issue, following these [contribution guidelines](https://openusd.org/release/contributing_to_usd.html). Don’t forget to link the PR back to this issue!

Thank you for being part of the USD community!

**Comment by pberto** [comment]
Can you at least please confirm if on linux you are still building with `python39` and with `ocio 2.1.3`?

**Comment by jesschimein** [comment]
Yes, that’s right. Python and OCIO are both part of the VFX platform and
we’re currently on VFX platform 2022.

**Comment by pberto** [comment]
FYI: I managed to make it work with that strict set of dependencies.
